src/NzbDrone.Core/Notifications/Signal/InvalidResponseException.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+
+namespace NzbDrone.Core.Notifications.Signal
+{
+    public class InvalidResponseException : Exception
+    {
+        public InvalidResponseException()
```

Comment:
Can we name this to something not so generic (preferably including `Signal` in the exception name)

Reply:
Sorry for the delay.

It's as in Telegram or pushover, the two other ones using this system

Reply:
i took into account your comments and modified my code.